backend/libs/parking/src/parking.service.spec.ts

```diff
@@ -87,7 +91,7 @@ describe('ParkingService', () => {
   });
 
   describe("assignParking", () => {
-      it("should return true if parking number is valid", async () => {
```

Comment:
Please actually run the tests locally before making the PR( run them using yarn test), it describes the errors. The reason everything is failing is because the tests are not passing. Also please don't have empty tests in the spec file.

Reply:
I noticed. I did not write the tests as you might know. The parking system is used by a lot of other subsystems so I wanted to get it out there, before getting to tests.
I wanted a review of my code structure from the rest of the team before wasting time on tests, as you can see will need to change as the actual functions change.
I can finish the tests before I merge that is fine, but it's going to be quite a wait. I am currently focused on getting things to actually work.

Reply:
Sounds good